modules/client/test/integration/client-addressList/methods.test.ts

```diff
@@ -134,11 +134,37 @@ describe("Client Address List", () => {
       const client = new ClientAddressList(ctxPlugin);
 
       const voteParams: IVoteProposalParams = {
-        pluginAddress: "0x1234567890123456789012345678901234567890",
-        proposalId: "0x1234567890123456789012345678901234567890",
+        pluginAddress,
+        proposalId: "0x0",
```

Comment:
how can it work with ID 0x0?

Reply:
The contracts accept a bigNumberish, this means `string | number | bigint | hex string| bignumber` so literally atm they dont care once we implement the new proposalID with validation this wont work and will need to be updated

Reply:
Ok, but as of today how can this work? Is the current proposalId 0x0 valid and open for vote?

If so, resolve and merge. I'm good for the rest :+1:

Reply:
Yes, its is created in the previous test